.github/workflows/takedown.yml

```diff
@@ -0,0 +1,33 @@
+name: Spam Detection
+
+on:
+  schedule:
+    - cron: '*/10 * * * *'  # run every ten minutes
```

Comment:
nit: we can use something like [`4-59/10 * * * *`](https://stackoverflow.com/questions/12786410/run-cron-job-every-n-minutes-plus-offset) to do cron job at X:04, X:14, X:24, X:34, X:44 and 54 to avoid the [high load issue](https://github.com/orgs/community/discussions/52477).

Reply:
I checked the action execution time and found that it didn't always start within the first minute of each ten-minute interval, but we can try `4-59/10 * * * *` to see if it works.
- [#7](https://github.com/cofacts/takedowns/actions/runs/12277263949/job/34256265687) Wed, 11 Dec 2024 13:22:02 GMT
- [#8](https://github.com/cofacts/takedowns/actions/runs/12277608982/job/34257379557) Wed, 11 Dec 2024 13:41:55 GMT
- [#9](https://github.com/cofacts/takedowns/actions/runs/12277774812/job/34257932373) Wed, 11 Dec 2024 13:51:48 GMT
- [#10](https://github.com/cofacts/takedowns/actions/runs/12278121530/job/34259081369) Wed, 11 Dec 2024 14:10:59 GMT